docs/src/configuration/services/_index.md

```diff
@@ -1,177 +1,182 @@
 {{< /note >}}
 
-### Type
+### Service options
 
-The `type` of your service. It's using the format `type:version`.
+The following table presents the keys you can define for each service:
 
-If you specify a version number which is not available, you'll see this error when pushing your changes:
+| Name            | Type        | Required          | Description |
+| --------------- | ----------- | ----------------- | ----------- |
+| `type`          | `string`    | Yes               | One of the [available services](#available-services) in the format `type:version`. |
+| `disk`          | `integer`   | For some services | The size in MB of the [persistent disk](#disk) allocated to the service. Can't be set for memory-resident-only services such as `memcache` and `redis`. |
+| `size`          | `string`    |                   | How many CPU and memory [resources to allocate](#size) to the service. Possible values are `AUTO` (default), `S`, `M`, `L`, `XL`, `2XL`, and `4XL`. |
+| `configuration` | dictionary  |                   | Some services have additional specific configuration options that can be defined here, such as specific endpoints. See the given service page for more details. |
 
-```bash
-Validating configuration files.
-E: Error parsing configuration files:
-    - services.mysql.type: 'mysql:5.6' is not a valid service type.
-```
+Both `disk` and `size` are limited by your plan settings.
 
-Service types and their supported versions include:
-<!--
-To update the versions in this table, use docs/data/registry.json
--->
-{{< readFile file="src/registry/images/tables/services_supported.md" markdownify="true">}}
+#### Disk
 
-### Disk
+Downsizing a service's persistent disk isn't currently supported
+in the `eu.platform.sh` and `us.platform.sh` regions.
+If necessary, [migrate your project](../../guides/general/region-migration.md).
 
-The `disk` attribute is the size of the persistent disk (in MB) allocated to the service.
+#### Size
 
-For example, the current default storage amount per project is 5GB (meaning 5120MB) which you can distribute between your application (as defined in `.platform.app.yaml`) and each of its services.  For memory-resident-only services such as `memcache` or `redis`, the `disk` key is not available and will generate an error if present.
+By default, Platform.sh allocates CPU and memory resources to each container automatically given the available resources on the plan.
+Some services are optimized for high CPU load, some for high memory load.
+If your plan is sufficiently large, you can customize the size of your service container.
```

Comment:
```suggestion
If your plan is sufficiently large and fits bigger containers, you can increase the size of your service container.
```
I'm trying to help, but I'm not sure it does, feel free to reword from here. The main goal is to spell that sufficiently large means that all containers need to fit in the plan 😬  might be okay as is though :)

Reply:
I saw the original idea as including the possibility of making a given service smaller. Like if your plan isn't super big and you want to give one service more room, you could maybe make service X smaller to make room for service Y.

But then the "sufficiently large" part makes less sense. :thinking: